transients-manager.php

```diff
@@ -51,6 +51,30 @@ class AM_Transients_Manager {
 	 */
 	public $next_cron_delete = 0;
 
+	/**
+         * ID of the transient
+	 * 
+  	 * @since 2.0.4
+    	 * @var int
+	 */
+	public $transient_id;
+
+	/**
+         * Transient object
+	 * 
+  	 * @since 2.0.4
+    	 * @var object
+	 */
+	public $transient;
+
+	/**
+         * Action name
+	 * 
+  	 * @since 2.0.4
+    	 * @var string
+	 */
```

Comment:
Looks like some whitespace issues.

Reply:
Thanks, @jtsternberg for the review.

Going to merge as is and fix these whitespaces in a follow-up commit.